internal/store/store.go

```diff
@@ -114,28 +114,11 @@ func New(stateDir *string) *Store {
 			log.Fatalf("cannot create output directory")
 		}
 
-		stateFile := *stateDir + "/state.json"
-		state, err := ioutil.ReadFile(stateFile)
-
-		if err == nil {
-			err := json.Unmarshal(state, &s)
-			if err != nil {
-				log.Fatalf("invalid initial state: %v", err)
-			}
-		} else if !os.IsNotExist(err) {
+		s.db = jsondb.New(*stateDir, 0600)
```

Comment:
Should this be passed in so it can be easily mocked?

Reply:
I was wondering about that and decided against it, because it would be a code change that's only beneficial to testing. This is something I'm trying to avoid.

Reply:
I've changed my mind on this after talking to @teg. It should be mocked, but it's out of scope for this PR. At least until we get all of the `stateDir` out of `store.go`.